models/modelling/olids/observations/int_person_frailty.sql

```
@@ -0,0 +1,46 @@
+with persons as (
+    select person_id from {{ ref('stg_aic_int_efi2_scores') }}
+    union
+    select person_id from {{ ref('int_rockwood_latest') }}
+    union
+    select person_id from {{ ref('fct_person_frailty_register') }}
+)
+
+select
+    p.person_id
+    -- AIC eFI2
+    , efi.efi_score as efi2_score
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)